database/mssql/scripts/dbscripts/PSP_PIMS_S25_01/Alter Up/89_DML_PIMS_PROPERTY_ID_SEQ.sql

```diff
@@ -0,0 +1,6 @@
+DECLARE @PIMS_PROPERTY_ID VARCHAR(MAX);
+DECLARE @sql VARCHAR(MAX);
+SET @PIMS_PROPERTY_ID = (SELECT max(PROPERTY_ID) from PIMS_PROPERTY) + 1;
+SET @sql = 'ALTER SEQUENCE PIMS_PROPERTY_ID_SEQ RESTART WITH ' + @PIMS_PROPERTY_ID
+EXEC (@sql)
+GO
```

Comment:
Is this necessary? Should the etl migration not do this instead? The problem is that since this is only executed on the s25 alter up, if another set of properties are added after this wont work

Reply:
This will allow IS 25.1 in UAT to go ahead. Future ETL work will need to add this script as well